b2sdk/api.py

```diff
@@ -423,14 +424,18 @@ def list_keys(self, start_application_key_id=None):
         )
 
     # other
-    def get_file_info(self, file_id):
+    def get_file_info(self, file_id: str) -> Dict[str, Any]:
```

Comment:
Just add a `get_file_info_by_id` method as alias here as everywhere else we use that kind of scheme

Reply:
Based on the last conversation, I was under the impression we are not doing this. Can you explain?

Reply:
Ok, you're right. Let's keep it then